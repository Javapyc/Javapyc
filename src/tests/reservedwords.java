class
public
static
extends
void
int
boolean
if
else
while
return
null
true
false
this
new
String
main
System.out.println
System.out.println(
ift
classpublic
this.bob
return;
if{}else{}
returnnull;
