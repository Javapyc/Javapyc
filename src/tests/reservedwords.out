ReservedWord, class
ReservedWord, public
ReservedWord, static
ReservedWord, extends
ReservedWord, void
ReservedWord, int
ReservedWord, boolean
ReservedWord, if
ReservedWord, else
ReservedWord, while
ReservedWord, return
ReservedWord, null
ReservedWord, true
ReservedWord, false
ReservedWord, this
ReservedWord, new
ReservedWord, String
ReservedWord, main
ReservedWord, System.out.println
ReservedWord, System.out.println
Delimiter, (
ID, ift
ID, classpublic
ReservedWord, this
Delimiter, .
ID, bob
ReservedWord, return
Delimiter, ;
ReservedWord, if
Delimiter, {
Delimiter, }
ReservedWord, else
Delimiter, {
Delimiter, }
ID, returnnull
Delimiter, ;
